app/vmselect/graphite/eval.go

```diff
@@ -155,8 +156,7 @@ func evalMetricExpr(ec *evalConfig, me *graphiteql.MetricExpr) (nextSeriesFunc,
 }
 
 func newNextSeriesForSearchQuery(ec *evalConfig, sq *storage.SearchQuery, expr graphiteql.Expr) (nextSeriesFunc, error) {
-	denyPartialResponse := true
-	rss, _, err := netstorage.ProcessSearchQuery(nil, denyPartialResponse, sq, ec.deadline)
+	rss, _, err := netstorage.ProcessSearchQuery(nil, ec.denyPartialResponse, sq, ec.deadline)
```

Comment:
This PR looks good overall. 

I'm not familiar with the graphite API. Do we need to receive the returned `isPartial` value and handle it like:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/d6415b2572d03f85ac5eed9cc3417d0a2900d3fe/app/vmselect/promql/eval.go#L1731

Reply:
It is impossible to propagate `isPartial` flag to graphite response - graphite render api returns array
https://graphite.readthedocs.io/en/latest/render_api.html#json